src/pom.xml

```diff
@@ -773,7 +770,7 @@
             <dependency>
                 <groupId>com.fasterxml.jackson.core</groupId>
                 <artifactId>jackson-databind</artifactId>
-                <version>${jackson.version}.1</version>
+                <version>${jackson.version}.3</version>
```

Comment:
They still didn't release a proper new version? I only added this sub-sub-sub-version because they only released this single artifact for a security fix and not the whole package...

Reply:
Yes... They did not yet release a proper new version ;-)